src/app.js

```diff
@@ -0,0 +1,92 @@
+import { clearAll, completeToDo } from './interact.js';
+
+const list = document.querySelector('.task-content');
+const clear = document.querySelector('.clear');
+
+let LIST = [];
+
+const addToDo = (toDo, id, done, trash) => {
```

Comment:
It would be very nice if you avoid a long number of arguments. Ideally, a function should have two or fewer arguments specified. The fewer the arguments, the easier is to test the function. For more information, you can check out [this link](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices).

Reply:
Thank you for your review, changes implemented